contracts/5/libs/EIP712BaseB.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity 0.5.13;
```

Comment:
Why this file is neede and why in it's name "B" is mentioned at the end?

Reply:
The EIP712 contract I wrote has a slightly different interface to the original EIP712Base.sol already written. I renamed mine to be similar to note the function of both is the same - but called it version "B" to avoid conflict. If chainId() isn't used in any contracts I've written I can remove BaseB - will check when able to